v1.0.0.yaml

```diff
@@ -2600,94 +2342,179 @@ paths:
+                    quality: 0
+                    density: 72
+                    hasTransparency: false
+                    pHash: f06830ca9f1e3e90
+                    exif:
+                      image:
+                        Make: Canon
+                        Model: Canon EOS 40D
+                        Orientation: 1
+                        XResolution: 72
+                        YResolution: 72
+                        ResolutionUnit: 2
+                        Software: GIMP 2.4.5
+                        ModifyDate: 2008:07:31 10:38:11
+                        YCbCrPositioning: 2
+                        ExifOffset: 214
+                        GPSInfo: 978
+                      thumbnail:
+                        Compression: 6
+                        XResolution: 72
+                        YResolution: 72
+                        ResolutionUnit: 2
+                        ThumbnailOffset: 1090
+                        ThumbnailLength: 1378
+                      exif:
+                        ExposureTime: 0.00625
+                        FNumber: 7.1
+                        ExposureProgram: 1
+                        ISO: 100
+                        ExifVersion: "0221"
+                        DateTimeOriginal: 2008:05:30 15:56:01
+                        CreateDate: 2008:05:30 15:56:01
+                        ShutterSpeedValue: 7.375
+                        ApertureValue: 5.625
+                        ExposureCompensation: 0
+                        MeteringMode: 5
+                        Flash: 9
+                        FocalLength: 135
+                        SubSecTime: "00"
+                        SubSecTimeOriginal: "00"
+                        SubSecTimeDigitized: "00"
+                        FlashpixVersion: "0100"
+                        ColorSpace: 1
+                        ExifImageWidth: 100
+                        ExifImageHeight: 68
+                        InteropOffset: 948
+                        FocalPlaneXResolution: 4438.356164383562
+                        FocalPlaneYResolution: 4445.969125214408
+                        FocalPlaneResolutionUnit: 2
+                        CustomRendered: 0
+                        ExposureMode: 1
+                        WhiteBalance: 0
+                        SceneCaptureType: 0
+                      gps:
+                        GPSVersionID:
+                          - 2
+                          - 2
+                          - 0
+                          - 0
+                      interoperability:
+                        InteropIndex: R98
+                        InteropVersion: "0100"
+                      makernote: {}
+                Video:
+                  value:
+                    height: 720
+                    width: 1280
+                    bitRate: 546524
+                    duration: 70
+                    audioCodec: aac
+                    videoCodec: h264
+                    size: 23490
+                    exif:
```

Comment:
Note: The `exif` key is not present in case of a video file. Only these keys are:
- "height"
- "width"
- "bitRate"
- "duration"
- "audioCodec"
- "videoCodec"
- "size"

Reply:
Also, as per the comment in upload-server, the exif tool only works for JPEG images.

See https://git.imagekit.io/search?group_id=12&project_id=73&search=%2F%2F+fails+when+format+is+not+JPEG+or+NO_EXIF_SEGMENT+found+-+no+action+needed

> Exif data isn't standard for video file formats but some camera manufacturers do add it forcibly.